swagger.yaml

```diff
@@ -4969,6 +4973,7 @@ definitions:
       - 'sufficient-network-latency-requirement-for-role'
       - 'sufficient-packet-loss-requirement-for-role'
       - 'has-default-route'
+      - 'domain-name-resolved-correctly'
```

Comment:
The validation is only for the API domain, so it should have the word `api` in the name

Reply:
Can also remove "correctly" - it's enough to say that it was resolved

Reply:
It doesn't, we're checking to see that it resolves to the exact VIP and not just resolve in general